grid-packages/ag-grid-docs/documentation/src/pages/components/quotes/quotesData.ts

```diff
@@ -0,0 +1,45 @@
+export interface QuotesDataItem {
+  name: string,
+  avatarUrl: string,
+  orgName: string,
+  orgIconUrl: string,
+  orgRole: string,
+  text: string,
+
+  /**
+    * The order to show on desktop. Leave empty to not show it.
```

Comment:
Since we don't have a separate mobile one:

```suggestion
    * The order to show. Leave empty to not show it.
```